requirements/dev.txt

```diff
@@ -131,7 +131,7 @@ packaging==20.9
     #   tox
 path.py==12.5.0
     # via edx-i18n-tools
-path==15.1.2
+path==16.0.0
```

Comment:
major version bump but no major change: https://github.com/jaraco/path/blob/main/CHANGES.rst#v1600 so it looks okay to go.